cranelift/assembler-x64/src/api.rs

```diff
@@ -24,17 +24,13 @@ pub trait CodeSink {
     /// required for assembling memory accesses.
     fn add_trap(&mut self, code: TrapCode);
 
-    /// Return the byte offset of the current location in the code buffer;
-    /// required for assembling RIP-relative memory accesses.
-    fn current_offset(&self) -> u32;
+    /// Inform the code buffer of a use of `target` is about to happen within
+    /// this buffer at the current offset. After this method is called bytes are
+    /// then placed at this offset.
```

Comment:
s/the bytes are then placed/the bytes are then expected to be placed/

(it's part of the contract for what the caller must do, rather than a result of this method call, right?)